packages/adminrouter/extra/dcos-adminrouter.service

```diff
@@ -23,6 +23,6 @@ ExecStartPre=/usr/bin/test -f /var/lib/dcos/cluster-id
 # Wait for auth backend to seed the secret key
 ExecStartPre=/usr/bin/curl --fail -sS -o /dev/null 127.0.0.1:8101/acs/api/v1/groups
 ExecStart=$PKG_PATH/nginx/sbin/nginx -c $PKG_PATH/nginx/conf/nginx.master.conf
-ExecReload=/usr/bin/kill -HUP $MAINPID
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
1. I checked the following distributions and confirmed that `/bin/kill` is present.

- ubuntu 16.04
- ubuntu 14.04
- ubuntu 12.04

- centos 7.3.1611
- centos 7.2.1511

- coreos 1298.1.0  (Alpha, Jan 22, 2017)
- coreos 1235.8.0 (Stable, Jan 31, 2017)
- coreos 522.6.0 (Stable, Jan 28, 2015)

- debian 8

Earlier debians seem to not ship whatever package provides /bin/kill in them by default.